LabExercise.ipynb

```diff
@@ -413,7 +413,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!git checkout origin/solutions -- boxblur.cu"
+    "!git show origin/solutions:boxblur.cu > solution.cu\n",
+    "!nvcc solution.cu -D EXERCISE_MODE=STARTING_CODE -o boxblur && ./boxblur\n",
+    "!nvcc solution.cu -D EXERCISE_MODE=EXERCISE_01 -o boxblur && ./boxblur\n",
+    "!nvcc solution.cu -D EXERCISE_MODE=EXERCISE_02 -o boxblur && ./boxblur\n",
+    "!nvcc solution.cu -D EXERCISE_MODE=EXERCISE_03 -o boxblur && ./boxblur"
```

Comment:
I added these compile and run commands so that it's easier for the users to compare the times against their own progress